projects/addon-mobile/directives/ripple/ripple.module.ts

```diff
@@ -1,9 +1,10 @@
 import {NgModule} from '@angular/core';
 
 import {TuiRippleDirective} from './ripple.directive';
+import {TuiRippleStylesComponent} from './ripple-styles.component';
 
 @NgModule({
-    declarations: [TuiRippleDirective],
+    declarations: [TuiRippleDirective, TuiRippleStylesComponent],
```

Comment:
Please add it to `entryComponents` since we support view engine at the moment.

Reply:
fixed